content/04.results.md

```diff
@@ -2,8 +2,22 @@
 
 ### Comparing bioRxiv to PubMed Central
 
-#### Global View
-1. Create a treemap visualization of top X terms that are different between bioRxiv and PubMed Central (based on odds ratio)
+#### Global Comparison of bioRxiv and PubMed Central 
+
+![
+BioRxiv has more category specific terms whereas PubMed Central has more patient oriented articles.
```

Comment:
Is this saying that there are different topic areas, or something else that I'm missing?